src/initialize-test-window.coffee

```diff
@@ -57,6 +57,12 @@ module.exports = ({blobStore}) ->
 
     document.title = "Spec Suite"
 
+    # Avoid throttling of test window by playing silence
+    context = new AudioContext()
+    source = context.createBufferSource()
+    source.connect(context.destination)
+    source.start(0)
+
     testRunner = require(testRunnerPath)
```

Comment:
I should note that as of Chrome 47, the visibility doesn't appear to be working to keep our tests running in the foreground. A new `--disable-background-timer-throttling` flag was [added](https://bugs.chromium.org/p/chromium/issues/detail?id=485425) in Chrome 49, but that's still in beta.